scripts/main/settings.js

```diff
@@ -457,32 +468,25 @@ settings.save_enter = function (e) {
 	// We only handle "enter"
 	if (e.which !== 13) return;
 
-	// show confirmation box
-	$(":focus").blur();
-
-	let action = {};
-	let cancel = {};
-
-	action.title = lychee.locale["ENTER"];
-	action.msg = lychee.html`<p style="color: #d92c34; font-size: 1.3em; font-weight: bold; text-transform: capitalize; text-align: center;">${lychee.locale["SAVE_RISK"]}</p>`;
-
-	cancel.title = lychee.locale["CANCEL"];
-
-	action.fn = function () {
-		settings.save(settings.getValues("#fullSettings"));
-		basicModal.close();
-	};
+	const saveSettingsConfirmationDialogBody =
+		'<p style="color: #d92c34; font-size: 1.3em; font-weight: bold; text-transform: capitalize; text-align: center;"></p>';
```

Comment:
```suggestion
		// TODO: move the style to the style file, where it belongs.
		'<p style="color: #d92c34; font-size: 1.3em; font-weight: bold; text-transform: capitalize; text-align: center;"></p>';
```

I know it was there before, but it's just too ugly to silently accept...

Reply:
Agreed.

I am even not sure why we want to increase the font size and make it bold. That is too much for my taste. IMHO, the "normal" warning dialog should be sufficient.